Packages/vcs/vcs/dv3d.py

```diff
@@ -156,8 +156,13 @@ def getConfigurationState(self, pname, **args):
         return self.cfgManager.getConfigurationState(pname, **args)
 
     def add_property(self, name):
-        fget = lambda self: self.getParameter(name)
-        fset = lambda self, value: self.setParameter(name, value)
+
+        def fget(self):
+            return self.getParameter(name)
```

Comment:
I had to update the flake8 test because I changed the directory path of the package, and then to make sure the update to the test worked, I had to run it. This uncovered a number of flake8 failures (due to some updates to flake8 that I guess we merged recently?).